charters/DataPortal/charter.md

```diff
@@ -0,0 +1,73 @@
+# Data Portal
+
+
+## Description
+The Data Portal is the landing site for the Human Cell Atlas (HCA) Data Coordination Platform (DCP). It is hosted at https://www.data.humancellatlas.org/ with an access link from the main HCA website at https://www.humancellatlas.org/. It provides users interested in the HCA with an easy and efficient way to explore and use data at cellular resolution.
```

Comment:
Is the url https://www.data.humancellatlas.org/ correct? I was sort of expecting https://data.humancellatlas.org. The currently deployed staging environment has no www; if it's the current deployment that's in error then this is fine.

Reply:
Good catch - now fixed.